test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -451,19 +451,43 @@ public void run()
             }
         });
     }
-    
+
     public static List<String> buildNodetoolArgs(List<String> args)
     {
         List<String> allArgs = new ArrayList<>();
         allArgs.add("bin/nodetool");
         allArgs.add("-p");
         allArgs.add(Integer.toString(jmxPort));
         allArgs.add("-h");
-        allArgs.add(jmxHost);
+        allArgs.add(jmxHost == null ? "127.0.0.1" : jmxHost);
```

Comment:
You can be testing the tool _without_ needing or wanting a server running. Like there's no need to start the server to test help messages i.e. Also you might want to test a tool's behavior when there is no reachable server, etc.

Reply:
makes sense, thanks.  In this case it may make more sense to also ignore `-h` but I will leave this comment as a "nit", so free to leave as is or change.